WeScan/Extensions/CGSize+Utils.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CGSize+Utils.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 17/2/2019.
+//  Copyright © 2019 WeTransfer. All rights reserved.
+//
+
+import Foundation
```

Comment:
if this file only meant for `CGSize` extension, how about importing individual declaration, like - 

`import struct CoreGraphics.CGSize`